backend/Tournament/views.py

```diff
@@ -0,0 +1,131 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth import authenticate
+from django.utils.decorators import method_decorator
+
+from .models import Tournament, \
+						TournamentPlayer
+from .serializers import TournamentSerializer, \
+	 						TournamentCreationSerializer, \
+							TournamentPlayerSerializer, \
+							TournamentInProgressSerializer
+from .managers import TournamentSetupManager, \
+						TournamentInProgressManager
+from .exceptions import TournamentSetupException, \
+							TournamentInProgressException
+from .tournamentState import TournamentState
+from User.models import User
+from shared_utilities.decorators import must_be_authenticated, \
+											must_not_be_username, \
+											check_that_valid_tournament_request
+
+import sys
+
+# Create your views here.
+class TournamentListView(APIView):
+	@method_decorator(must_be_authenticated)
+	def post(self, request):
+		body_data = request.data.copy()
+		body_data['host_user'] = request.user.id
+
+		tournament_creation_serializer = TournamentCreationSerializer(data=body_data)
+		if not tournament_creation_serializer.is_valid():
+			return Response({'error': 'invalid request for tournament creation'}, status=status.HTTP_400_BAD_REQUEST)
+		
+		try:
+			tournament = TournamentSetupManager.create_tournament_and_tournament_player_for_host(tournament_creation_serializer.validated_data)
+			host_tournament_player_serializer = TournamentPlayerSerializer(tournament.players.all().first())
+			return Response({
+				'tournament_id': tournament.id,
+				'tournament_player': host_tournament_player_serializer.data
```

Comment:
You should do like we do with the User and other things too, an Input and Output Serializers that expose these fields rather than creating it here by hand.